pkg/config/config.go

```diff
@@ -97,6 +110,11 @@ var defaultServerConfig = &ServerConfig{
 		},
 	},
 	ReadHeaderTimeoutSeconds: 32, // just shy of requestTimeoutUpperBound
+	KubeClientConfig: KubeClientConfig{
+		QPS:     100,
```

Comment:
nit: do you want to keep these defaults in sync with the RESTClient defaults or are these more sensible/preferred?

Reply:
By RESTClient defaults do you mean the regular k8s defaults? I'm not sure I understand, but if so it could be useful. The only place these will come in to play is the speed that users can create workflow CRDs. A user was using a workflow which internally started 100 launchplans and found that they are throttled pretty aggressively.

Reply:
I mean the ones in the comment where you define the `KubeClientConfig`- not a big deal either way